docker/scripts/functions.sh

```diff
@@ -788,17 +788,20 @@ function perc_reset() {
 	fi
 
 	#Check/set personality
-	if perccli64 /c0 show personality | grep "Current Personality" | grep "HBA-Mode" >/dev/null; then
-		echo "PERCCLI - Controller in HBA-Mode - OK"
+	if perccli64 /c0 show personality | grep "Current Personality" | grep -E "eHBA|HBA-Mode" >/dev/null; then
+		echo "PERCCLI - Controller in HBA/eHBA-Mode - OK"
 	elif [[ $percmodel == 'PERCH710PMini' || $percmodel == 'PERCH740PMini' ]]; then
 		echo "PERCCLI - Skipping set HBA-Mode. This $percmodel does not support HBA mode"
+	elif [[ $percmodel == 'PERCH745Front' ]]; then
+		echo "PERCCLI - Setting personality to eHBA-Mode"
+		perccli64 /c0 set personality=eHBA
 	else
 		echo "PERCCLI - Setting personality to HBA-Mode"
 		perccli64 /c0 set personality=HBA
 	fi
 
-	#Check/delete all VDs!
-	if perccli64 /c0 /vall show | grep "No VDs" >/dev/null; then
+	#Check/delete all VDs! -- VDs and VD's it seems
```

Comment:
This comment is going to seem very cryptic to someone unless they dig into the related PR. Perhaps keep it as is?

Reply:
https://github.com/tinkerbell/osie/pull/249/commits/f69d8922a8957c2a096a723845f96abaebc1a011 reverts to the original for clarity